pages/dkp/kommander/2.2/install/mgmt-cluster-apps/index.md

```diff
@@ -11,16 +11,16 @@ This section only details requirements for management cluster-specific applicati
 
 This table describes the workspace platform applications specific to the management cluster, minimum resource requirements, and minimum persistent storage requirements.
 
-| Name                      | Minimum Resources Suggested    | Minimum Persistent Storage Required |
-| ------------------------- | ------------------------------ | ----------------------------------- |
-| centralized-grafana       | cpu: 200m<br />memory: 100Mi   |                                     |
-| centralized-kubecost      | cpu: 1200m<br />memory: 4151Mi | # of PVs: 1<br />PV sizes: 32Gi     |
-| chartmuseum               |                                | # of PVs: 1<br />PV sizes: 2Gi      |
-| dex                       | cpu: 100m<br />memory: 50Mi    |                                     |
-| dex-k8s-authenticator     | cpu: 100m<br />memory: 128Mi   |                                     |
-| gitea                     | cpu: 500m<br />memory: 512Mi   | # of PVs: 2<br />PV sizes: 10Gi     |
-| karma                     |                                |                                     |
-| kommander-flux            | cpu: 4000m<br />memory: 4Gi    |                                     |
-| kubefed                   | cpu: 300m<br />memory: 192Mi   |                                     |
-| thanos                    |                                |                                     |
-| traefik-forward-auth-mgmt | cpu: 100m<br />memory: 128Mi   |                                     |
+| Common Name       | Application Name          | Minimum Resources Suggested | Minimum Persistent Storage Required |
+|-------------------|---------------------------|-----------------------------|-------------------------------------|
+| Grafana           | centralized-grafana       | cpu: 200m memory: 100Mi     |                                     |
```

Comment:
We need to keep the `<br>` tags for the line breaks. I'll push a commit to add these back

Reply:
done in https://github.com/mesosphere/dcos-docs-site/pull/4390/commits/2552f07c2c0239f13bec16605292ff80267e02d7